apps/election-manager/src/lib/votecounting.ts

```diff
@@ -448,22 +449,21 @@ function getVotingMethodForCastVoteRecord(CVR: CastVoteRecord): VotingMethod {
 
 function getTallyForCastVoteRecords(
   election: Election,
-  castVoteRecords: CastVoteRecord[],
+  castVoteRecords: Dictionary<CastVoteRecord>,
   filterContestsByParty?: string
 ): Tally {
   const allVotes: VotesDict[] = []
-  const cvrFiles: CastVoteRecord[] = []
   const ballotCountsByVotingMethod: Dictionary<number> = {}
   Object.values(VotingMethod).forEach(
     (votingMethod) => (ballotCountsByVotingMethod[votingMethod] = 0)
   )
-  castVoteRecords.forEach((CVR) => {
+  for (const cvrId in castVoteRecords) {
```

Comment:
Do we want to have a guard here, i.e. `if (Object.prototype.hasOwnProperty.call(castVoteRecords, cvrId)) {`? I noticed elsewhere you had `Object.entries`. Seems like we're kinda wishing for a `Map`?